src/components/PageElements/PageElements.tsx

```diff
@@ -67,7 +68,20 @@ const PageElements: React.FC<PageElementProps> = ({
               currentResponse: responsesByCode?.[element.code],
             }
             // Wrapper displays response & changes requested warning for LOQ re-submission
-            return <ApplicationViewWrapper key={`question_${element.code}`} {...props} />
+            return (
+              <div className="form-element-wrapper" key={`question_${element.code}`}>
```

Comment:
hmm, wondering if this should be in ApplicationViewWrapper or here ?

I can see that it would complicate `ApplicationViewWrapper` return statement

I originally thought that the changes required border would be outside of ApplicationViewWrapper too. I guess we can see if it needs a refactor in the future.

Reply:
Yeah, originally it was outside `ApplicationViewWrapper`, but the spacing between elements pushed the "Changes Required" comments too far away from the field they relate too. Worked better having them inside it, and I kind of think it works if everything related to one element sits within one `<Form.Field>`.

But yeah, can maybe refactor later if we want.